napari/_vispy/vendored/volume.py

```diff
@@ -201,6 +201,106 @@
     return final_color;
 }}
 
+vec4 calculateCategoricalColor(vec4 betterColor, vec3 loc, vec3 step)
```

Comment:
why is this `betterColor` not just `color`?

Reply:
I copied the API for the `calculateColor` function above. I agree it's not the best name, but I thought maybe it was better to stay consistent.